command.go

```diff
@@ -121,6 +124,15 @@ func UnRegister(topic string, channel string) *Command {
 	return &Command{[]byte("UNREGISTER"), params, nil}
 }
 
+// SyncState creates a new Command to sync a topic/channel state changes, like paused, unpaused
+func SyncState(topic string, channel string, state []byte) *Command {
```

Comment:
I would expect this function to accept `state` as a `map[string]interface{}`, rather than `[]byte` and this function would handle serialization (like `Identify`'s implementation above).

Reply:
done, good suggestion